sinks/api/supported_metrics.go

```diff
@@ -255,7 +256,50 @@ var statMetrics = []SupportedStatMetric{
 		},
 		OnlyExportIfChanged: true,
 	},
-
+	{
+		MetricDescriptor: MetricDescriptor{
+			Name:        "cpu/node_usage",
+			Description: "Cumulative CPU usage on all cores on a node",
+			Type:        MetricCumulative,
+			ValueType:   ValueInt64,
+			Units:       UnitsNanoseconds,
+		},
+		HasValue: func(spec *source_api.ContainerSpec) bool {
+			return spec.HasCpu && spec.HasResourceId
```

Comment:
The heapster core is in a bit of a flux. I apologize for the inconvenience. 
We currently push node usage as part of `cpu/usage` metric, with `container_name` label set to `machine`. Will that suffice your use case? 
If not, this change will not expose node_usage if resourceId is not set, which might be the case in baremetal deployments. Should we instead annotate ContainerSpec with either a Container name or isNode boolean and use that here?
If resource ID is available we can push that label as part of node_usage.